docs/MachineConfiguration.md

```diff
@@ -56,6 +56,31 @@ type MachineConfigSpec struct {
 }
 ```
 
+The actual custom resource manifest then could look like this:
+
+```
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  name: worker-2864988432
+spec:
+  config:
+    ignition:
+      version: 2.2.0
+    storage:
+      files:
+      - contents:
+          source: data:,%20
+        filesystem: root
+        mode: 384
+        path: /root/myfile
+  osImageURL: quay.io/openshift/rhcos:v4.0.1
+```
+
+(Notice how it's the usual ignition config object *inplace*, not as a JSON
```

Comment:
> Notice how it's the usual ignition config object *inplace*, not as a JSON

The struct above clearly declares that `config` field is IgnitionConfig object and not a string... ?

> also note the `config` and `osImageURL` casing follows the `json:` markers in the definition above

That is standard practice in Kubernetes objects.

Reply:
in particular, we are including these notes because they are things that tripped us up when we were testing out the daemon. I think it's useful to point out these things in the docs even if they are standard practice elsewhere. 

> The struct above clearly declares that config field is IgnitionConfig object and not a string... ?

it can be difficult to predict exactly how something is going to get parsed when you aren't working with this stuff all the time. 

> That is standard practice in Kubernetes objects.

this is specifically being pointed out for people who aren't necessarily familiar with standard kubernetes practices.

Reply:
> also note the `config` and `osImageURL` casing follows the `json:`
markers in the definition above

Can you expand this to cover the ignitionconfig obect too? because it applied to ignitionconfig object too.